corehq/apps/hqwebapp/management/commands/resource_static.py

```diff
@@ -4,35 +4,20 @@
 
 from django.conf import settings
 from django.contrib.staticfiles import finders
-from django.core import cache
 from django.core.management.base import BaseCommand
 
 import yaml
 
 from dimagi.utils import gitinfo
 
-rcache = cache.caches['redis']
-RESOURCE_PREFIX = '#resource_%s'
-
 
 class Command(BaseCommand):
-    help = "Prints the paths of all the static files"
+    help = """
+        Hashes the contents of all static files and stores the results in resource_versions.yaml.
+    """
 
     root_dir = settings.FILEPATH
 
-    def add_arguments(self, parser):
-        parser.add_argument(
-            '--clear',
-            action='store_true',
-            default=False,
-        )
-
-    def current_sha(self):
-        current_snapshot = gitinfo.get_project_snapshot(self.root_dir, submodules=False)
-        sha = current_snapshot['commits'][0]['sha']
-        print("Current commit SHA: %s" % sha)
-        return sha
-
     def output_resources(self, resources, overwrite=True, path=None):
```

Comment:
Is path ever used?

Reply:
Yes, it's there for tests: https://github.com/dimagi/commcare-hq/blob/1455305a9d5ad2c1da2b560df8e2590463ccee09/corehq/apps/hqwebapp/tests/test_static_resource_command.py#L44